### DISCUSSION TITLE
directus apply template doesn't work

### DISCUSSION THREAD

**Discussion by aldoprogra** [directus-labs/agency-os#108] [Q&A]
On local works, but now that I have the postgres db on railway, not only is really slow the loading off collections, but eventually will not complete successfully. Tables are correctly created, but seems there is no content inside.

End with this error:

Applying template - AgencyOS to http://0.0.0.0:8055
Loading 76 collections and 700 fields.... !
    TypeError: Cannot read properties of undefined (reading 'status')

**Comment by bryantgillespie** [comment]
Are you trying to load into a local instance? Or one hosted on Railway?
Because you mentioned Railway but the error you included shows you are loading into http://0.0.0.0:8055.

What does your config look like?

**Comment by aldoprogra** [comment]
yes it is a local instance. General speaking I am having troubles deploying the instance, if I use the template works but in this way I have no control over the env. So my strategy is to bootstrap over a deployed db and then deploy my docker file of directus. Without the env file auth doesn't work, I suppose because of CORS false, but is a different issue.
attached my env. Do you think I should deploy the instance before apply the template?
[env.txt](https://github.com/directus-labs/agency-os/files/14835662/env.txt)

**Comment by aldoprogra** [comment]
update, even with directus and postgres deployed still same error: 
Loading 49 files... !
    TypeError: Cannot read properties of undefined (reading 
    'status')

**Reply by sethbrasile** [reply]
Same problem here. Fresh directus instance, used `docker-compose.yml` from repo with no changes, followed instructions in readme, running commands local to the docker host (amd64 linux) and got to this point:

```
Logged in as Admin User
------------------
Applying template - AgencyOS to http://localhost:8055
Loading 76 collections and 700 fields.... done
Loaded collections and fields.
Loading 176 relations... done
Loaded relations
Loading 4 roles... done
Loaded roles
Loading 17 folders... done
Loaded folders
 ›   Warning: Status 503 • "Service \"files\" is unavailable. Couldn't save file 08177527-b867-47e9-b5d5-eab146b3cdba.jpg."
 ›
 ›   Warning: Status 503 • "Service \"files\" is unavailable. Couldn't save file 08242513-ca8f-471f-b69b-82703a19d1e5.svg."
 ›
 ›   Warning: Status 503 • "Service \"files\" is unavailable. Couldn't save file 0dfe66dd-ceba-4dca-8907-03b2dda49443.svg."
 ›
 ›   Warning: Status 503 • "Service \"files\" is unavailable. Couldn't save file 12e02b82-b4a4-4aaf-8ca4-e73c20a41c26.jpeg."
 ›
Loading 49 files... !
    TypeError: Cannot read properties of undefined (reading 'status')
```

**Comment by JSDA123** [comment]
I was able to successfully install the template by beginning fresh with Directus v10.11, applying the template, then upgrading sequentially to the latest version.  This way the template is applied when upgrading to Directus v11 which includes significant changes to the schema.  

This is easy if using docker.  Start with image directus/directus:10.11, then upgrade the image tag until you are at directus/directus:latest
